managed_vms/django_tutorial/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django==1.9
+MySQL-python==1.2.5
```

Comment:
Use `mysqlclient` instead, as it supports Python 3. [source](https://docs.djangoproject.com/en/1.9/ref/databases/#mysql-db-api-drivers)